tests/selenium/test-runner.js

```diff
@@ -1,8 +1,22 @@
+var server = require('../../server');
```

Comment:
The handling of next is basically the same as before, cept now I plug into certain points and do driver related stuff.